lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1816,14 +1787,8 @@ export class UserAgentApplication {
           if (hashParams.hasOwnProperty(Constants.idToken)) {
             response.idToken = new IdToken(hashParams[Constants.idToken]);
           } else {
-            response.idToken = new IdToken(this.cacheStorage.getItem(Constants.idTokenKey));
```

Comment:
Is this a merge issue? or why it was removed?

Reply:
Not removed. Extracted into the responseWithIdToken function.